js/src/data/reducer.js

```diff
@@ -124,6 +124,35 @@ const reducer = ( state = DEFAULT_STATE, action ) => {
 			return setIn( state, 'mc.shipping.rates', action.shippingRates );
 		}
 
+		case TYPES.UPSERT_SHIPPING_RATES: {
```

Comment:
💅 I think maybe it would be better to add back the related reducer tests that were deleted in #1262

Reply:
Oh yes! Done in [`c5b9818` (#1313)](https://github.com/woocommerce/google-listings-and-ads/pull/1313/commits/c5b9818e4a3f7ee919c20ff15e60efac7a8beab0).